django/contrib/gis/static/gis/js/OLMapWidget.js

```diff
@@ -1,378 +1,385 @@
 /*global OpenLayers*/
-/*eslint indent:1*/
```

Comment:
Nope, just 4 spaces. I assume I added this to suppress warnings before the file indentation is fixed.

@timgraham btw excellent job making all those 1s into 2s in the `.eslintrc` file! :+1: :tada: